holidays/constants.py

```diff
@@ -9,12 +9,32 @@
 #  Website: https://github.com/dr-prodigy/python-holidays
 #  License: MIT (see LICENSE file)
 
-HOLIDAY_NAME_DELIMITER = "; "  # Holiday names separator.
+# flake8: noqa: F401
 
-MON, TUE, WED, THU, FRI, SAT, SUN = range(7)
-WEEKEND = (SAT, SUN)
+from holidays.calendars.gregorian import (
```

Comment:
Why are we keeping here these constants import? For external compatibility?

Reply:
That's right, it needs to stay backward compatible